boards/hail/src/main.rs

```diff
@@ -127,6 +133,8 @@ unsafe fn set_pin_primary_functions() {
     PA[05].configure(Some(A)); // A1 - ADC1
     PA[06].configure(Some(A)); // DAC
     PA[07].configure(None); //... WKP - Wakeup
+                            // PA[06].configure(Some(E)); // ACAN0 - ACIFC
+                            // PA[07].configure(Some(E)); // ACAP0 - ACIFC
```

Comment:
These comments are good, but I don't think we should have a syscall exposed that no apps can actually used (no assigned pins).

Reply:
I think the example usage in the capsule is enough. Also, if we can figure out shared components then it would be super easy to add in if a user wants to.

Reply:
Done and done! Let me know if you have any other feedback